### PR TITLE
Tezos answers

### PR DESCRIPTION
**Pull request by RichardRed0x**

### CODE REVIEW

File: answers/Tezos.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+* *How are changes to the consensus rules decided on and implemented?*
+
+Bakers participate in a four-stage voting [process](https://medium.com/tezos/amending-tezos-b77949d97e1e) to adopt changes to the consensus rules. The amendment process has 4 phases, each lasting 8 "cycles" (a cycle lasts around 3 days, so each phase lasts for around 24 days). In the first phase bakers submit and upvote proposals. When the first phase ends, the most upvoted proposal will progress to phase 2, where it must be approved by at least 80% of bakers. If the criteria are met, this is followed by a testing phase in which a testnet fork with the changes applied is created and runs for 48 hours (a further testnet matching the proposal may be run for the rest of this phase to allow further testing). After the testing phase, bakers vote on whether the changes should be activated, with an 80% supermajority required (adjusts dynamically based on previous participation rates). After this 4th phase the changes are activated (or not) and the cycle begins again with new proposals.
```
Comment (s-ben) [inline answers/Tezos.md]: One remaining question is how, technically, consensus rules are updated. Tezos' language seems to imply it works as it does in PoW, with nodes voluntarily updating, but according to davecgh updates are accomplished via a master key, defeating the whole purpose. Would be nice to know how it actually happens. Perhaps the voting can veto changes made via master key?
https://matrix.to/#/!kdpEDksmOMNrlMqffD:decred.org/$155206988021084WhHYu:decred.org?via=decred.org&via=matrix.org

https://matrix.to/#/!kdpEDksmOMNrlMqffD:decred.org/$155207103621108FyNeQ:decred.org?via=decred.org&via=matrix.org

File: answers/Tezos.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+### Further reading 
+
+<https://tezos.gitlab.io/mainnet/>
+
+
+
+# Funding
+
+There is only one [implementation](https://gitlab.com/tezos/tezos/) of a Tezos full/baking node.
```
Comment (s-ben) [inline answers/Tezos.md]: Move to Blockchain Governance section?

File: answers/Tezos.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+There is very little information available about how the Foundation is allocating its funds. There are [often](https://tezos.foundation/news/tezzigator-receives-grant-to-expand-cloud-hsm-support-for-tezos-bakers) [news](https://tezos.foundation/news/supporting-the-development-of-block-explorers-and-smart-contract-languages-for-the-tezos-ecosystem) [items](https://tezos.foundation/news/tezos-foundation-supports-tezos-riot-forum) on the foundation website that say a particular team/project is being funded, but these have no information about how much funding is being given. 
+
+The foundation is being [audited](https://tezos.foundation/news/statement-on-2017-statutory-audit-completed-by-pwc-switzerland) by PwC but no details of its finances are being openly published.
```
Comment (s-ben) [inline answers/Tezos.md]: @Hanon's [reply](https://matrix.to/#/!lbzTjhzNbIaDbuAxkS:decred.org/$155791174413926zvbMo:decred.org?via=decred.org&via=matrix.org) in writing channel mentions the Swiss Federal Supervisory Authority. Worth mentioning here?

> s_ben: I wrote this about Tezos: "there is a Tezos Foundation that relies on a Council and the Swiss Federal Supervisory Authority to ensure that the Foundation’s assets are used responsibly and PwC Switzerland to audit the Foundation’s finances and operations. Grants are administered by the Tocqueville Group and all proposals are evaluated by an advisory committee of qualified community members, although the Council has the final vote."

**Comment by RichardRed0x** [comment]
Thanks for the comments.

**Pull request merged**